shop/src/main/java/ro/msg/learning/shop/ShopApplication.java

```diff
@@ -10,4 +14,11 @@ public static void main(String[] args) {
 		SpringApplication.run(ShopApplication.class, args);
 	}
 
+	@Bean
+	protected CorsConfigurationSource corsConfigurationSource() {
```

Comment:
Just take care that in a real application the CORS should be limited only to a specific set of URLs. Also, this configuration should be in a separate config file.